eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody.hs

```diff
@@ -533,85 +533,53 @@ pattern BabbageTxBody ::
-    txnetworkid
+  { btbInputs,
+    btbCollateral,
+    btbReferenceInputs,
+    btbOutputs,
+    btbCollateralReturn,
+    btbTotalCollateral,
+    btbCerts,
+    btbWdrls,
+    btbTxFee,
+    btbValidityInterval,
+    btbUpdate,
+    btbReqSignerHashes,
+    btbMint,
+    btbScriptIntegrityHash,
+    btbAuxDataHash,
+    btbTxNetworkId
   } <-
   TxBodyConstr
     ( Memo
-        TxBodyRaw
-          { _spendInputs = inputs,
-            _collateralInputs = collateral,
-            _referenceInputs = referenceInputs,
-            _outputs = outputs,
-            _collateralReturn = collateralReturn,
-            _totalCollateral = totalCollateral,
-            _certs = txcerts,
-            _wdrls = txwdrls,
-            _txfee = txfee,
-            _vldt = txvldt,
-            _update = txUpdates,
-            _reqSignerHashes = reqSignerHashes,
-            _mint = mint,
-            _scriptIntegrityHash = scriptIntegrityHash,
-            _adHash = adHash,
-            _txnetworkid = txnetworkid
+        BabbageTxBodyRaw
+          { btbrSpendInputs = btbInputs,
+            btbrCollateralInputs = btbCollateral,
+            btbrReferenceInputs = btbReferenceInputs,
+            btbrOutputs = btbOutputs,
+            btbrCollateralReturn = btbCollateralReturn,
+            btbrTotalCollateral = btbTotalCollateral,
+            btbrCerts = btbCerts,
+            btbrWdrls = btbWdrls,
+            btbrTxFee = btbTxFee,
+            btbrValidityInterval = btbValidityInterval,
+            btbrUpdate = btbUpdate,
+            btbrReqSignerHashes = btbReqSignerHashes,
+            btbrMint = btbMint,
+            btbrScriptIntegrityHash = btbScriptIntegrityHash,
+            btbrAuxDataHash = btbAuxDataHash,
+            btbrTxNetworkId = btbTxNetworkId
           }
         _
       )
   where
-    BabbageTxBody
-      inputsX
-      collateralX
-      referenceInputsX
-      outputsX
-      collateralReturnX
-      totalCollateralX
-      certsX
-      wdrlsX
-      txfeeX
-      vldtX
-      updateX
-      reqSignerHashesX
-      mintX
-      scriptIntegrityHashX
-      adHashX
-      txnetworkidX =
-        mkBabbageTxBodyFromRaw $
-          TxBodyRaw
-            inputsX
-            collateralX
-            referenceInputsX
-            outputsX
-            collateralReturnX
-            totalCollateralX
-            certsX
-            wdrlsX
-            txfeeX
-            vldtX
-            updateX
-            reqSignerHashesX
-            mintX
-            scriptIntegrityHashX
-            adHashX
-            txnetworkidX
+    BabbageTxBody inputs collateral referenceInputs outputs collateralReturn totalCollateral certs wdrls txfee vldt update reqSignerHashes mint scriptIntegrityHash adHash txnetworkid =
+      mkBabbageTxBodyFromRaw $
+        BabbageTxBodyRaw inputs collateral referenceInputs outputs collateralReturn totalCollateral certs wdrls txfee vldt update reqSignerHashes mint scriptIntegrityHash adHash txnetworkid
```

Comment:
Same here with respect to record style syntax

Reply:
Noted! All `[Era]TxBody` pattern synonyms should need the switch :+1: